recipes/LibriSpeech/ASR/CTC/train_with_wav2vec_k2.py

```diff
@@ -351,9 +351,9 @@ def text_pipeline(wrd):
         },
     )
 
-    run_on_main(
```

Comment:
Isn't `run_on_main` used to perform the an operation on the master node of a multi-gpu/multi-node experiment?

Reply:
Yes, it is true. However, if you look inside the `download_file` function [here](https://speechbrain.readthedocs.io/en/latest/_modules/speechbrain/utils/data_utils.html#download_file), which is called in our recipe to download files, there is a function `sb.utils.distributed.ddp_barrier()` called, which, I guess, might have made our multi-GPU DDP training fail. 

If you run our recipe now, you will find that, even though we are not using `run_on_main` to download files, we only download the files only by the main process.